src/styles/index.css

```diff
@@ -1,9 +1,8 @@
-body {
-  margin: 0;
-  padding: 0;
-  font-family: sans-serif;
-}
+* { margin: 0; padding: 0; font-family: sans-serif; }
+.App, #router, #root { height: 100%; }
```

Comment:
@bill-feng I wasn't sure why `.App`'s height is 100%. Could you explain it?

Reply:
we have a .App component and unless its height is set to 100%, the property is not propagated to any components it contains..